python/hail/tests/test_io.py

```diff
@@ -75,3 +75,78 @@ def test_call_fields(self):
         entries = entries.key_by('locus', 's')
         entries = entries.select('GT', 'GTA', 'GTZ')
         self.assertTrue(entries._same(expected))
+        
+class BGENTests(unittest.TestCase):
```

Comment:
I have a PR  (#3691) that refactors these tests as well. Please don't fix the other import types: VCF, PLINK.